docs/05_Installing_Kubernetes_Control_Plane.md

```diff
@@ -102,21 +102,24 @@ for that purpose.
 
 ### Common variables
 
-Let's define some reusable shell variables to use throughout this chapter:
+Let's define some reusable shell variables to use throughout this chapter. Run
+`ip a` to identify network device in the vmaddr line below:
 
 ```bash
 arch=amd64
 
 etcd_version=3.5.15
 k8s_version=1.31.0
 
-vmaddr=$(ip addr show enp0s1 | grep -Po 'inet \K192\.168\.1\.\d+')
+vmaddr=$(ip addr show enp0s2 | grep -Po 'inet \K192\.168\.3\.\d+')
 vmname=$(hostname -s)
 ```
 
 ### Installing `etcd`
 
-Let's download the `etcd` binary, unpack it and copy into appropriate system directory:
+Let's download the `etcd` binary, unpack it and copy into appropriate system directory.
+You might need to add `nameserver 8.8.8.8` at the end of `/etc/resolv.conf` if there are
+resolution issues below:
```

Comment:
Can we also add an entry `server=8.8.8.8` to `/etc/dnsmasq.conf` for a fallback dns? If a worker node needs to pull nginx image from docker hub, it really doesn't have an authoritative dns server for that? It will try dnsmasq but its only aware of the entries in `/etc/hosts`

Am i missing something here?

Reply:
Shouldn't `dnsmasq` (the DNS server, not DHCP) simply fall back to the host machine's DNS when it doesn't have an entry in `/etc/hosts`? I haven't tested the guide on Linux recently, but it was always my impression that this is how `dnsmasq` works. I certainly didn't have any issues with VM's access to public internet.

Reply:
I'll see what i uncover this weekend 😄

Reply:
I retested the guide on a fresh Debian installation and I had no issues with DNS resolution on VMs.